### ISSUE TITLE
Embedded flutterview can be seen but doesnt recognize clicks

### ISSUE THREAD

**Issue by wrumble** [flutter/flutter#12765]
## Steps to Reproduce

Firstly apologies for the terrible Folder tree i couldnt get it to work with just spaces.

So, I have semi-successfully managed to embed a flutter view inside a collectionView in one of my existing iOS projects. The steps involved were fairly simple.

After following flutter install instructions I then 

I updated my pod file to contain and call the following function:
```
        def addFlutter
            pod 'Flutter', :path => "~/flutter/bin/cache/artifacts/engine/ios"

            if File.exists? '../.flutter-plugins'
              flutter_root = File.expand_path('..')
              File.foreach('../.flutter-plugins') { |line|
                plugin = line.split(pattern='=')
                if plugin.length == 2
                  name = plugin[0].strip()
                  path = plugin[1].strip()
                  resolved_path = File.expand_path("#{path}/ios", flutter_root)
                  pod name, :path => resolved_path
                else
                  puts "Invalid plugin specification: #{line}"
                end
              }
            end
          end
```

After a `pod install` i could successfully `import Flutter` into my appDelegate and it can now inherit from `FlutterDelegate` instead of `UIResponder, UIApplicationDelegate` 

     class AppDelegate: FlutterAppDelegate {

Then in my viewController i declare my flutterViewController

 `private let flutterViewController = FlutterViewController()`

and add its to my viewControllers subview

`view.addSubview(flutterViewController.view)`

From here I then created a parent folder with the same name as my iosProject to move my current iosProject into.

Current tree:

/iosProject
---|
-----/iosProject
--------|
---------xcworkspace etc

i then cd to above the parent folder.
run `flutter create iosProject`
This adds all the usual files into the parent folder as it has the same name as the iosProject
Current tree:

/iosProject
---|
----/iosProject
--------|
--------xcworkspace etc
----/android
----/assets
----/lib
----/test
----/ios
----android.iml
----iosProject.iml
----iosProject_android.iml
----pubspec.lock
----pubspec.yaml
----.idea
----.packages 

From here I took the `app.flx`, `App.framework` and `Flutter.framework` from the flutter_view project example you have supplied that are created on running and copied them into a new Flutter folder i have created inside my iosProject folder. I then also copied across `AppFrameworkInfo.plist`, `Debug.xcconfig`, `Generated.xcconfig` and `Release.xcconfig` from the generated ios folder in my parent folder. I then deleted the ios folder.

/iosProject
----|
----/iosProject
--------|
--------/App.framework
--------/Flutter.framework
---------app.flx
---------AppFrameworkInfo.plist
--------Debug.xcconfig
--------Generated.xcconfig
--------Release.xcconfig
--------xcworkspace etc
----/android
----/assets
----/lib
----/test
----android.iml
----iosProject.iml
----iosProject_android.iml
----pubspec.lock
----pubspec.yaml
----.idea
----.packages 

Now if I open and run Xcode the app builds and I can see the flutter view where it counts how many times you have tapped the button but tapping the `+` button does nothing.

## Logs

`flutter run -d c03c63487c83d10558370847be241056932f40fb` 

`Launching lib/main.dart on iPhone in debug mode...
Found saved certificate choice "iPhone Developer: Wayne Rumble (XXXXXXXX)". To clear, use "flutter config".
Signing iOS app for device deployment using developer identity: "iPhone Developer: Wayne Rumble (GH738DW7EX)"
Running Xcode build...                               65.9s
Failed to build iOS app
Error output from Xcode build:
↳
    2017-10-27 16:44:42.546 xcodebuild[36379:3327892]  iPhoneConnect: ## Unable to mount developer disk image, (Error Domain=com.apple.dtdevicekit Code=601 "Could not locate device support files." UserInfo={NSLocalizedRecoverySuggestion=This iPhone 5s (Model A1457, A1518, A1528, A1530) is running iOS 11.1 (15B93), which may not be supported by this version of Xcode., DeviceType=iPhone6,2, NSLocalizedDescription=Could not locate device support files.}) {
        DeviceType = "iPhone6,2";
        NSLocalizedDescription = "Could not locate device support files.";
        NSLocalizedRecoverySuggestion = "This iPhone 5s (Model A1457, A1518, A1528, A1530) is running iOS 11.1 (15B93), which may not be supported by this version of Xcode.";
    }
    ** BUILD FAILED **

    The following build commands failed:
    	CopyPlistFile /Users/waynerumble/Desktop/splitter/build/ios/Debug-iphoneos/Runner.app/AppFrameworkInfo.plist Flutter/AppFrameworkInfo.plist
    	CpResource Flutter/Debug.xcconfig /Users/waynerumble/Desktop/splitter/build/ios/Debug-iphoneos/Runner.app/Debug.xcconfig
    (2 failures)
Xcode's output:

`

There is a lot more output from xcode here can post it if needed.

There are some xcode errors but the main thing id like to point out is the final errors where it is trying to run the original Runner.app, how can i create a flutter project where the ios app is called whatever i want?

`flutter analyze`

`Analyzing /Users/waynerumble/Desktop/splitter...
No issues found!
Ran in 21.7s`

lol

## Flutter Doctor

`flutter doctor`

`[✓] Flutter (on Mac OS X 10.12.6 16G29, locale en-GB, channel alpha)
    • Flutter at /Users/waynerumble/Desktop/flutter
    • Framework revision e8aa40eddd (10 days ago), 2017-10-17 15:42:40 -0700
    • Engine revision 7c4142808c
    • Tools Dart version 1.25.0-dev.11.0

[✓] Android toolchain - develop for Android devices (Android SDK 26.0.2)
    • Android SDK at /Users/waynerumble/Library/Android/sdk
    • Platform android-26, build-tools 26.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] iOS toolchain - develop for iOS devices (Xcode 9.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.0.1, Build version 9A1004
    • ios-deploy 1.9.2
    • CocoaPods version 1.3.1

[✓] Android Studio (version 2.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Connected devices
    • iPhone   • c03c63487c83d10558370847be241056932f40fb • ios • iOS 11.1
    • iPhone 7 • 476E9E8C-8248-4EF6-8054-67900D603E83     • ios • iOS 11.0 (simulator)`

**Comment by realcarlos** [comment]
yes, there is no a guidebook to show how to embed a flutter view into a exsiting iOS project.

**Comment by Hixie** [comment]
cc @mravn-google

**Comment by mravn-google** [comment]
https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps

**Comment by jamesderlin** [comment]
Is this still a problem?  In the parent `UIViewController`, did you also do:

```Swift
addChildViewController(flutterViewController)
view.addSubview(flutterViewController.view)
flutterViewController.didMove(toParentViewController: self)
```
?  I am able to embed and interact with a child FlutterView when I tried it.
